src/renderer/viz/expressions/aggregation/cluster/ClusterAggregation.js

```diff
@@ -42,7 +41,9 @@ export default class ClusterAggregation extends BaseExpression {
         checkType(this._expressionName, 'property', 0, this.type, this.property);
     }
 
-    _resolveAliases () {}
+    _resolveAliases (aliases) {
```

Comment:
I guess we can just remove this method, cause this replicates the BaseExpression implementation:
```this._getChildren().map(child => child._resolveAliases(aliases));``` (without the return!)